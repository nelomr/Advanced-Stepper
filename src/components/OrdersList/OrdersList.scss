.orders-page {
    flex: 1;
    width: 100%;
    margin-bottom: -4rem;
    background-color: $background-color;
}

.orders {
    min-height: 445px;
    max-width: 840px;
    margin: 50px auto;
    background-color: $light-color;
}

.orders--title {
    padding: 30px;
    margin-bottom: 30px;
    text-transform: uppercase;
    font-weight: bold;
    font-size: 18px;
    text-align: left;
    border-bottom: 1px solid $grey-medium;
}

.orders-list {
    padding: 0 0 40px;

    @include breakpoint($min-tablet) {
        padding: 0 50px 40px;
    }
}
