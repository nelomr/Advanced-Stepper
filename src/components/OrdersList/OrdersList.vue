<template>
    <div class="orders">
        <div class="orders--title">Informaci√≥n de tus pedidos</div>
        <div class="orders-list">
            <slot></slot>
        </div>
    </div>
</template>

<script>
export default {
  name: 'order-list'
}
</script>

<style lang="scss">
    @import "./OrdersList.scss";
</style>