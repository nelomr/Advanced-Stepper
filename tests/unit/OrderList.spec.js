import {shallowMount} from '@vue/test-utils';
import OrdersList from '@/components/OrdersList/OrdersList';

describe('OrdersList', () => {
    const build = () => {
        const wrapper = shallowMount(OrdersList, {});

        return {
            wrapper
        };
    };

    it('renders the component', () => {
        const {wrapper} = build();

        expect(wrapper.html()).toMatchSnapshot();
    });
});
