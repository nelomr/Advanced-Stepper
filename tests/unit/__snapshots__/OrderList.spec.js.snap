// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersList renders the component 1`] = `
<div class="orders">
  <div class="orders--title">Informaci√≥n de tus pedidos</div>
  <div class="orders-list"></div>
</div>
`;
